packages/plugin-betswirl/src/actions/info.ts

```diff
@@ -0,0 +1,86 @@
+import {
+    type Action,
+    type IAgentRuntime,
+    type Memory,
+    type HandlerCallback,
+    type State,
+    type ActionExample,
+} from "@moxie-protocol/core";
+
+export const infoAction: Action = {
+    name: "BETSWIRL_INFO",
+    similes: [
+        "WHAT_IS_BETSWIRL",
+        "HOW_DOES_BETSWIRL_WORK",
+        "HOW_TO_PLAY_BETSWIRL",
+        "WHAT_IS_PLAY_GAMES_OF_LUCK",
+        "HOW_DOES_PLAY_GAMES_OF_LUCK_WORK",
+        "HOW_TO_PLAY_GAMES_OF_LUCK",
+    ],
+    description:
+        "BetSwirl's skills offers you onchain casino games: Dice, Coin Toss and Roulette. Once you placed the bet, a randomness is drawn from Chainlink VRF and used to resolve the game, and you immediately receive your payout in your wallet if you win. If you want more details, visit https://www.betswirl.com.",
+    suppressInitialMessage: true,
+    validate: async (
+        _runtime: IAgentRuntime,
+        _message: Memory,
+        _state: State
+    ) => true,
+    handler: async (
+        runtime: IAgentRuntime,
+        message: Memory,
+        state: State,
+        _options: { [key: string]: unknown },
+        callback: HandlerCallback
+    ) => {
+        try {
+            const casinoTokens = await getCasinoTokens();
+            const tokenSymbols = casinoTokens.map((token) => token.symbol);
+            const tokenSymbolsString = tokenSymbols.join(", ");
+            await callback({
+                text: `Play Games Of Luck/BetSwirl Skills offers you to play onchain casino games on Base:
```

Comment:
Just Betswirl to avoid confusion
```suggestion
                text: `BetSwirl Skills offers you to play onchain casino games on Base:
```